expected/wasm32-wasi/predefined-macros.txt

```diff
@@ -2026,18 +2026,18 @@
 #define YESSTR 0x50002
 #define YXDOMAIN ns_r_yxdomain
 #define YXRRSET ns_r_yxrrset
-#define _ALLOCA_H 
```

Comment:
Why idea whats happening with all these whitespace changes?

Reply:
Sure - the line endings were inconsistent, intermixing Linux (LF) and Windows (CR LF) newlines. Additionally a few lines had spaces at the end of lines.
I've decided to leave it in, but I can revert if you prefer :)

Reply:
Would you mind landing the no-op update of this file separately?

Reply:
This file is entirely auto-generated, and should remain so, so that when people do builds and generate new versions of this file, we can compare with with `diff`. The easiest way to keep it up to date when making changes is to just copy your newly generated file into the tree, rather than manually editing it.

Reply:
Indeed, I was assuming that it was in an inconsistent state previously causing a larger-than-expected diff when it was copied.    Hense the suggestion of a no-op re-built and copy prior to landing.  But indeed, if you are hand-editing that is the wrong approach.